docs/lexicon.html

```diff
@@ -240,6 +240,61 @@ <h3><a name="canonicalise">canonicalise</a></h3>
       </ul>
     </p>
 
+    <h3><a name="exec">exec</a></h3>
+
+    <p><pre class="rule"><code>exec(cmd, stdout=True, stderr=False)</code></pre></p>
+
+    <p>Executes a command and returns the output as a string if the command's
+      exit code returns <code>0</code>. If <code>stdout</code>
```

Comment:
We might want to mention that the spaces are trimmed here (to save someone an annoying debugging journey if they ever wanted it :) )